src/rgw/rgw_file.cc

```diff
@@ -1942,7 +1942,7 @@ namespace rgw {
       cs_info.blocks = std::move(compressor->get_compression_blocks());
       encode(cs_info, tmp);
       attrs[RGW_ATTR_COMPRESSION] = tmp;
-      ldout(state->cct, 20) << "storing " << RGW_ATTR_COMPRESSION
+      ldpp_dout(this, 20) << "storing " << RGW_ATTR_COMPRESSION
```

Comment:
@cbodley `RGWWriteRequest` is a `RGWPutObj`, should we remove the dpp in this function call or leave it?

Reply:
remove